requirements/common.txt

```diff
@@ -1,4 +1,5 @@
 argcomplete==1.11.0
+boto3==1.17.36
```

Comment:
@ehofesmann FYI you need to include any new dependencies in `setup.py` as well in order for them to be automatically installed when someone installs the package from pypi.

The `requirements.txt` files just document the exact versions of packages that we use when developing, for reproducibility when working with source installs.